redirects.txt

```diff
@@ -315,4 +315,7 @@ developer/reference/javascript/qweb.rst                          developer/refer
 developer/reference/backend/assets.rst                           developer/reference/frontend/assets.rst
 
 developer/reference/frontend/owl_component_system.rst            developer/reference/frontend/owl_components.rst
-developer/reference/frontend/generic_components.rst              developer/reference/frontend/owl_components.rst
\ No newline at end of file
+developer/reference/frontend/generic_components.rst              developer/reference/frontend/owl_components.rst
+
+developer/misc/i18n/localization.rst                    developer/howto/localization.rst
+developer/misc/i18n/translations.rst                    developer/howto/translations.rst
```

Comment:
the move of the translations file does not really enter the scope of this PR (and should at least be in a separate commit)